interface/ceed-operator.c

```diff
@@ -93,8 +93,8 @@ int CeedOperatorCreate(Ceed ceed, CeedQFunction qf, CeedQFunction dqf,
   @ref Basic
 **/
 int CeedOperatorSetField(CeedOperator op, const char *fieldname,
-                         CeedElemRestriction r, CeedBasis b,
-                         CeedVector v) {
+                         CeedElemRestriction r, CeedTransposeMode lmode,
```

Comment:
Documentation above needs to be updated to include and explain the new parameter.

Reply:
Updated